### PR TITLE
Switch from forcetk to force.js

### PR DESCRIPTION
**Pull request by wmathurin**
All tests / sample apps updated.
NB: the modified force.js should probably also be published to https://github.com/ccoenraets/forcejs

PR for iOS and Android repos will come next.

### CODE REVIEW

**Comment by wmathurin** [comment]
Has anyone had a chance to review this? Thanks.

File: libs/force.js [hunk 1]
```diff
@@ -1,7 +1,33 @@
...
+ * HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+ * NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ *
+ * Based on: https://github.com/ccoenraets/forcejs/blob/a9348b90a046e12aac9b56864359c24e5b246abe/force.js
+ *
  * ForceJS - REST toolkit for Salesforce.com
  * Author: Christophe Coenraets @ccoenraets
- * Version: 0.6
+ * Version: 1.0
```
Comment (bhariharan) [inline libs/force.js]: Should we call this library `force.mobilesdk.js` to indicate it's a version specific to `Mobile SDK`?

File: libs/force.js [hunk 2]
```diff
@@ -51,9 +80,119 @@ var force = (function () {
...
+
+    // Testing only
+       requestHandler;
+        
+
+
+    /**
+     * Compute SalesforceMobileSDK for web app.
+     */
+    function computeWebAppSdkAgent(navigatorUserAgent) {
```
Comment (bhariharan) [inline libs/force.js]: Do we need this? If we're using native network stack, the user agent should get tagged on every REST request that goes out. Is this specifically for the browser XHR requests?

File: libs/force.js [hunk 3]
```diff
@@ -51,9 +80,119 @@ var force = (function () {
...
+            var match = /Mac OS X ([0-9_]*)/.exec(navigatorUserAgent);
+            return (match != null && match.length == 2 ? match[1].replace(/_/g, ".") : "Unknown");
+        };
+
+        var getWindowsVersion = function() {
+            var match = /Windows NT ([0-9\.]*)/.exec(navigatorUserAgent);
+            return (match != null && match.length == 2 ? match[1] : "Unknown");
+        };
+
+        var match = /(iPhone|iPad|iPod|Android|Windows Phone|Macintosh|Windows)/.exec(navigatorUserAgent);
```
Comment (bhariharan) [inline libs/force.js]: Do we still need `iPod`? ;-)

File: libs/force.js (hunk 1 above)
Comment (wmathurin) [inline libs/force.js]: All the trails etc out there refer to force.js so I don't want to rename the file.
Instead I was thinking of pushing the modified force.js to https://github.com/ccoenraets/forcejs as well to make sure everybody uses the same one.

Comment (bhariharan) [inline libs/force.js]: Sounds good!

**Comment by bhariharan** [comment]
:+1

File: libs/force.js (hunk 2 above)
Comment (wmathurin) [inline libs/force.js]: There are web apps that use smartsync.js and we wanted to keep track of those.

File: libs/force.js (hunk 3 above)
Comment (wmathurin) [inline libs/force.js]: Probably not.

**Pull request merged**